_posts/2015-06-03-team-offsite.md

```diff
@@ -0,0 +1,31 @@
+---
+layout: post
+title: "Team ePages 6 offsite event"
+date: "2015-06-03 13:44:00"
+icon: users
+tags: team event offsite
+categories: events
+authors: ["Fouad-Steffen Nabhan"]
+---
+
+Working hard and embrace the nature? How does that fit together? Our ePages 6 development team organised a team offsite event and convened in a lovely castle to focus on knowledge transfer. What did they do? Read on!
+
+We found this nice place called "Die Burg Schöna" right in the last corner of Saxon Switzerland, Germany to come together some hours away from the office to approach a tight agenda of topics in our team.
+
+Surrounded by nature...
+
+{% image blog-offsite-landscape.jpg %}{% endimage %}
+
+...we discovered the benefits of team work on more than one occasion. It helped find some well hidden caches, to manage some obstacles during our team event as well as to get the information we needed to flow freely amongst all team members.
+
+{% image blog-offsite-work.jpg %}{% endimage %}
+
+We managed to bring all team members to a basic level of understanding concerning all topics on the agenda. Due to the limited time, we could not go too much into detail, so we still have a to-do list to go through in the next couple of months.
+
+To gather new strength inbetween the workshops, we went hiking and filled our lungs with oxygen and let our brains rest a little while. This pretty much improved communication as well as team spirit.
```

Comment:
"We managed to bring all team members to a basic level of understanding" sounds a bit as people don't know what is going on. Maybe rather something like "We discussed... and updated each other"